pull.go

```diff
@@ -167,14 +167,14 @@ func pullImage(ctx context.Context, store storage.Store, imageName string, optio
 	}()
 
 	logrus.Debugf("copying %q to %q", spec, destName)
-	err = cp.Image(ctx, policyContext, destRef, srcRef, getCopyOptions(options.ReportWriter, options.SystemContext, nil, ""))
+	err = cp.Image(ctx, policyContext, destRef, srcRef, getCopyOptions(options.ReportWriter, sc, nil, ""))
```

Comment:
In new.go:newBuilder, we tease out the systemContext from options and then pass it down to here without using it prior that I can see.  Could we just drop systemContext as a param to several functions and just pull the value out from options here in this code?  No a change necessary for this PR, but maybe a touchup issue for later.

Reply:
We'd have to call `getSystemContext()` to override the default signature policy path with the configured one for each case, but that's not a deal-breaker.

Reply:
yeah, and signaturepolicy should be in the the options too.